### DISCUSSION TITLE
When mergeProps merges with null it becomes null.  Is that correct?

### DISCUSSION THREAD

**Discussion by rostero1** [adobe/react-spectrum#3041] [Q&A] [answered]
**Requested:**

`mergeProps({onBlur: () => {}}, {onBlur: null })`

**Expected:**

`{onBlur: () => {}}`

**Got:**

`{onBlur: null}`

**Comment by snowystinger** [comment, marked as answer]
That's coming from https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/utils/src/mergeProps.ts#L45
where we check that the two props we are merging are functions.

We could probably ignore null/undefined for props with the name prefix `on`. We could just `continue` so we don't add them in.
